pkgs/servers/web-apps/phppgadmin/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub } :
+
+stdenv.mkDerivation rec {
+  pname = "phppgadmin";
+  version = "7.13.0";
+
+  src = fetchFromGitHub {
+    owner  = "phppgadmin";
+    repo   = "phppgadmin";
+    rev    = "REL_${builtins.replaceStrings [ "." ] [ "-" ] version}";
+    sha256 = "13bnnidzp0q7c6jz7vcfamfi884h8q654n3x3f2b2clfpng0cc0v";
+  };
+
+  phases = [ "installPhase" ];
```

Comment:
```suggestion
```

Please disable phases you do not want to run. Overwriting phases can break hooks silently.

Reply:
now i get what you meant. thanks for the explanation.